dev_guide/managing_images.adoc

```diff
@@ -918,6 +925,19 @@ After the secret is configured, proceed with creating the new image stream or
 using the `oc import-image` command. During the import process, {product-title}
 will pick up the secrets and provide them to the remote party.
 
+[[trusting-registries]]
+=== Adding Trusted Certificates for External Registries
+
+If the registry you are importing from is using a certificate which is not signed
+by a standard certificate authority, you will need to explicitly configure the
+system to trust the registry's certificate or signing authority.  This can be 
+done by adding the CA certificate or registry certificate to the host system
+running the registry import controller (typically the master node).
+
+The certificate or CA certificate must be added to `/etc/pki/tls/certs` or `/etc/pki/ca-trust`,
+respectively, on the host system.
```

Comment:
It would be worth to mention that `update-ca-trust` needs to be run on RH and similar systems. Whether the restart of marster/nodes is necessary and whether this applies to pods interacting with the registries.

Reply:
This is only for image-import, so i would assume non-master nodes and pods are unaffected. (There is other doc that needs to be done for those things, of course).

As for whether a master restart is required, honestly I don't know, but i think the answer is no, i think the CAs are reread on each client request setup.

Reply:
updated w/ the update-ca-trust command